.gitignore

```diff
@@ -10,4 +10,5 @@ certs/
 ipa-ucdavis-edu.env
 cypress.env.json
 cypress/videos
-cypress/screenshots
\ No newline at end of file
+cypress/screenshots
```

Comment:
Do not update .gitignore in this PR. Make a separate PR which suits that purpose if you want .gitignore updated.